dags/bqetl_search.py

```diff
@@ -70,6 +70,7 @@
         email=["bewu@mozilla.com", "frank@mozilla.com", "telemetry-alerts@mozilla.com"],
         date_partition_parameter="submission_date",
         depends_on_past=False,
+        allow_field_addition_dates=("2020-07-13",),
```

Comment:
There's probably not much value to keeping old dates in this list, and in fact it would be slightly safer to remove old dates so that we don't accidentally allow date addition in backfills. So how about we change this field to a single date `allow_field_addition_on_date` and expect it to be overwritten for each new schema update.